src/views/GovernanceReviewTeam/index.test.tsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { MemoryRouter } from 'react-router-dom';
+import { mount } from 'enzyme';
+import configureMockStore from 'redux-mock-store';
+
+import { initialSystemIntakeForm } from 'data/systemIntake';
+import { closedIntakeStatuses, openIntakeStatuses } from 'types/systemIntake';
+
+import GovernanceReviewTeam from './index';
+
+jest.mock('@okta/okta-react', () => ({
+  useOktaAuth: () => {
+    return {
+      authState: {
+        isAuthenticated: true
+      },
+      authService: {
+        getAccessToken: () => Promise.resolve('test-access-token'),
+        getUser: () =>
+          Promise.resolve({
+            name: 'John Doe'
+          })
+      }
+    };
+  }
+}));
+
+describe('The GRT Review page', () => {
+  it('shows open status', () => {
+    const mockStore = configureMockStore();
+    const store = mockStore({
+      systemIntake: {
+        systemIntake: {
+          ...initialSystemIntakeForm,
+          status:
+            openIntakeStatuses[
```

Comment:
Take a random pick at one of the open statuses

Reply:
This is worrisome because it means our tests are different every time, which means if we have flaky results here, it will be nearly impossible to track down. I would pick one status here, and we should maybe make code modifications elsewhere to make sure the status map is well-tested.